scripts/buildCSS.js

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env node
+
+const fs = require('fs');
+const CleanCSS = require('clean-css');
+
+const CSSInterface = require('react-with-styles-interface-css').default;
+const registerMaxSpecificity = require('react-with-styles-interface-css').registerMaxSpecificity;
+const compileCSS = require('react-with-styles-interface-css/compile').default;
+
+const registerInterfaceWithDefaultTheme = require('../src/utils/registerInterfaceWithDefaultTheme').default;
+
+const args = process.argv.slice(2);
+const optimizeForProduction = args.includes('-o') || args.includes('--optimize');
+
+require('../test/_helpers/ignoreSVGStrings');
+
+registerMaxSpecificity(11);
+registerInterfaceWithDefaultTheme(CSSInterface);
+
+const DateRangePickerPath = './src/components/DateRangePicker.jsx';
```

Comment:
don't you want to pull in the babel output, not the raw code?

Reply:
I don't... think so. At least, that's not what the css interface uses as its tests.